fetch.bs

```diff
@@ -4766,7 +4779,10 @@ these steps:
 
      <dt>"<code>manual</code>"
      <dd><p>Set <var>response</var> to an <a>opaque-redirect filtered response</a> whose
-     <a for="filtered response">internal response</a> is <var>actualResponse</var>.
+     <a for="filtered response">internal response</a> is <var>actualResponse</var>, and set
+     <var>fetchParams</var>'s <a for="fetch params">controller</a>'s <a
+     for="fetch controller">next redirect steps</a> to run <a>HTTP-redirect fetch</a> given
+     <var>fetchParams</var> and <var>response</var>.
```

Comment:
We should only do this when mode is "navigate".

Reply:
Done